tests/SecurityDetails/SecurityDetailsConnector.test.ts

```diff
@@ -390,3 +390,29 @@ export async function MetaLoad (
         'Meta table should have multi row structure.'
     );
 }
+
+export async function bondStyleBoxBreakdownLoad (
+    api: MC.Shared.MorningstarAPIOptions
+) {
+    const connector = new MC.SecurityDetailsConnector({
+        api,
+        security: {
+            id: 'F00001GPCX',
+            idType: 'MSID'
+        },
+        converter: {
+            type: 'BondStyleBoxBreakdown'
+        }
+    });
+
+    await connector.load();
+
+    Assert.deepStrictEqual(
+        connector.table.getColumnNames(),
+        [
+            'BondStyleBoxBreakdown_Type',
+            'BondStyleBoxBreakdown_SalePosition_N'
+        ],
+        'Bond Style Box Breakdown table should exist of expected columns.'
+    );
```

Comment:
Shall we check if ANY data is returned and parsed? I mean we know that there will be 9 categories, because the matrix is 3x3. I think we can ignore category=99.

Reply:
Hmm, how about checking if the length of the values array is `>=9` ?

Reply:
Sounds good, let's try it